packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -163,12 +179,15 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
       _state = _DragState.accepted;
       final Offset delta = _pendingDragOffset;
       final Duration timestamp = _lastPendingEventTimestamp;
+      final Offset startPosition = dragStartBehavior == DragStartBehavior.start
```

Comment:
please use a switch rather than `?:`. The analyzer will then tell us if we ever add a value to the enum.